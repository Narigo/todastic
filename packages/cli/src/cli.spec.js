const { execSync } = require("child_process");

describe("command line interface", () => {
  it("can track time", async () => {
    expect(
      execSync(`node ${__dirname}/cli.js track ${__dirname}/__tests__/filter-sample.todo`).toString()
    ).toMatchSnapshot();
  });
});
