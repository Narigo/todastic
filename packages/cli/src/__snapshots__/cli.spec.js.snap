// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`command line interface can track time 1`] = `
"Tracked time: 03:30:00
"
`;
